api/client/src/main/java/jakarta/websocket/ClientEndpointConfig.java

```diff
@@ -64,6 +64,8 @@ public interface ClientEndpointConfig extends EndpointConfig {
      *
      * @return the SSLContext to use to establish a secure connection to the server or {@code null} if an insecure
      *         connection should be established
+     *
+     * @since WebSocket 2.1
```

Comment:
Perhaps it should be ...

``` java
* @since Jakarta WebSocket 2.1
```

so that it isn't confused with one of the WebSocket specs outside of Jakarta EE?

Reply:
I went for consistency with the existing @since tags

Reply:
Its a minor nit, just trying to be as clear as possible in the javadoc.
I'm totally onboard with being consistent with past usage too.